mercury.spec

```
@@ -8,19 +8,18 @@
 %global shortmchecksum_commit %%(c=%%{mchecksum_commit}; echo ${c:0:7})
 
 %bcond_with use_release
-%global dl_version 2.0.0a1
+%global dl_version 2.0.0rc1
 
 Name: mercury
-Version: 2.0.0~a1
-Release: 2.git.%{shortmercury_commit}%{?dist}
+Version: 2.0.0rc1
+Release: 1.git.%{shortmercury_commit}%{?dist}
```

Since this is an actual upstream [pre-]release, no git hash in the Release:
```suggestion
Release: 1%{?dist}
```

Ultimately, we should make this conditional on whether `mercury_commit` is defined and just not define it when doing an actual upstream release.

Although, going forward, I hope we can remain happy with their official upstream release, which they do often, and in between releases we apply a patch or two.